build/builder/Dockerfile

```diff
@@ -28,7 +28,6 @@ COPY operator/ operator/
 COPY third_party/ third_party/
 COPY scripts/generate-third-party-licenses scripts/generate-third-party-licenses
 COPY scripts/resource-autogen scripts/resource-autogen
-COPY version.txt third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/fwtransport/version.txt
```

Comment:
Is this intended? 
If so, I guess we can delete third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/fwtransport/version.txt (below)

Reply:
oh, ah! Didn't pay attention to the file name, I was thinking that this updates the third_party dir.